src/styles/main.scss

```diff
@@ -0,0 +1,60 @@
+$height_watch: 80vmin;
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: auto;
+}
+
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $height_watch;
+  height: $height_watch;
```

Comment:
I guess variable should be named 'watch_size' as it's not only height but width too.
By the way, you could use a mixin for setting same width and height, as you set the identical width and height in watch__round too
https://sass-lang.com/documentation/at-rules/mixin

Reply:
done